apl-core/src/test/resources/shard-last-hashes-null.csv

```diff
@@ -0,0 +1,3 @@
+SHARD_ID(-5|19|0),SHARD_HASH(-3|2147483647|0),SHARD_HEIGHT(4|10|0),ZIP_HASH_CRC(-3|2147483647|0),PRUNABLE_ZIP_HASH(-3|2147483647|0),GENERATOR_IDS(2003|0|0),BLOCK_TIMEOUTS(2003|0|0),BLOCK_TIMESTAMPS(2003|0|0)
```

Comment:
Is file name correct ?? **shard-last-hashes-null.csv**

Reply:
File name is correct